src/codegen/queries.rs

```diff
@@ -596,7 +631,7 @@ fn gen_query_module(module: &PreparedModule, config: &Config) -> proc_macro2::To
 
     for row in module.rows.values() {
         if row.is_named {
-            let row_tokens = gen_row_structs(row, &config.types.derive_traits, &ctx);
+            let row_tokens = gen_row_structs(row, &config.types.derive_traits, &ctx, config);
```

Comment:
nitpick: i'd remove the `config.types.derive_traits` param to only have `config` and get the `derive_traits` from it in the function.